packages/assistant-stream/src/core/tool/ToolExecutionStream.ts

```
@@ -83,21 +83,24 @@ export class ToolExecutionStream extends PipeableTransformStream<
               if (chunk.meta.type !== "tool-call") break;
 
               const { toolCallId, toolName } = chunk.meta;
+              const streamController = toolCallControllers.get(toolCallId)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)